packages/replayio/src/commands/remove.ts

```diff
@@ -15,6 +15,10 @@ registerCommand("remove")
   .action(remove);
 
 async function remove(shortIds: string[], { all = false }: { all?: boolean }) {
+  if (all) {
+    removeAllFromDisk();
+  }
```

Comment:
This now gets called prematurely in the sense that it leads to a weird experience. It removes everything from the disk, then reads the recordings log and since there is no recording log at this point this function logs "No recordings found". I think this should be fine-tuned.

Previously we were printing a list of everything that was removed. I'm on the fence when it comes to how useful that was. On one hand the user was notified about what was removed, on the other hand... they really couldn't quite act on it anyhow as far as I know. Printing a potentially huge table of redundant data doesn't seem that useful so perhaps a simple confirmation of what just happened would be enough here

Reply:
> It removes everything from the disk, then reads the recordings log and since there is no recording log at this point this function logs "No recordings found"

D'oh!